src/ape/managers/compilers.py

```diff
@@ -110,6 +110,14 @@ def compile(self, contract_filepaths: List[Path]) -> Dict[str, ContractType]:
 
         return contract_types_dict  # type: ignore
 
+    def get_compiler_settings(self, contract_filepaths: List[Path], base_path: Optional[Path]):
+        settings = dict()
+        for compiler in self.registered_compilers:
+            settings[compiler] = self.registered_compilers[compiler].get_compiler_settings(
+                contract_filepaths, base_path=base_path
+            )
+        return settings
```

Comment:
Be a little more performant
```suggestion
        return {
            compiler: self.registered_compilers[compiler].get_compiler_settings(
                contract_filepaths, base_path=base_path
            )
            for compiler in self.registered_compilers
        }
```